andela/Ah-frontend-guardians
# #162685622 Run CodeClimate Test Coverage

#### What does this PR do?
This PR runs the codeclimate test coverage.

#### Description of Task to be completed?
The task involves creating a .circleci directory to which a config.yml is added.
Furthermore, the circle ci is connected to the repo.

#### How should this be manually tested?
Deploy your branch to the repo and check out the link below:
https://circleci.com/gh/andela/Ah-frontend-guardians


#### What are the relevant pivotal tracker stories?
[#162685622](https://www.pivotaltracker.com/story/show/162685622)
